integration/container/exec_test.go

```diff
@@ -1,13 +1,20 @@
 package container // import "github.com/docker/docker/integration/container"
 
 import (
+	"encoding/json"
 	"io"
+	"net/http"
+	"net/url"
+	"runtime"
 	"strings"
 	"testing"
 	"time"
 
+	"github.com/docker/docker/api/types"
 	containertypes "github.com/docker/docker/api/types/container"
+	"github.com/docker/docker/errdefs"
 	"github.com/docker/docker/integration/internal/container"
+	req "github.com/docker/docker/testutil/request"
```

Comment:
Do we need the `req` alias?

Reply:
Oh, we don't; looks like this was because I copy/pasted bits in this file from the other one.


I'm working on a follow-up; I'll fix it up for both files there